IdentityCore/src/cache/MSIDToken.m

```diff
@@ -204,6 +285,34 @@ - (instancetype)initWithTokenResponse:(MSIDTokenResponse *)response
     return self;
 }
 
+#pragma mark - Private
+
+- (NSMutableOrderedSet<NSString *> *)scopesFromString:(NSString *)scopesString
```

Comment:
this method is available in NSString+MSIDExtensions category (scopeSet), can we reuse that one?

Reply:
done